api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -104,9 +105,34 @@ public void updateCompletionTime(
   public List<AccessModuleStatus> getClientAccessModuleStatus(DbUser user) {
     return userAccessModuleDao.getAllByUser(user).stream()
         .map(a -> userAccessModuleMapper.dbToModule(a, getExpirationTime(a).orElse(null)))
+        .filter(a -> isModuleEnabledInEnvironment(a.getModuleName()))
         .collect(Collectors.toList());
   }
 
+  @Override
+  public boolean isModuleCompliant(DbUser dbUser, AccessModuleName accessModuleName) {
```

Comment:
Please add a check for `isModuleEnabledInEnvironment` in here - if the module is **not** enabled, the user is **always** compliant

Reply:
also added a test